pkg/capabilities/v2/protoc/pkg/pb/cre_metadata.proto

```diff
@@ -16,3 +16,11 @@ message CapabilityMetadata {
 extend google.protobuf.ServiceOptions {
   CapabilityMetadata capability = 50000; // 50000-99999 is the range for custom options.
 }
+
+message CapabilityMethodMetadata {
+  bool map_to_untyped_trigger_api = 1;
```

Comment:
Can you call this map to legacy and plug in into both trigger and action?

Reply:
moved it to 'map_to_untyped_api'